packages/js/src/executors/release-publish/release-publish.impl.ts

```diff
@@ -32,12 +34,12 @@ export default async function runExecutor(
   const projectConfig =
     context.projectsConfigurations!.projects[context.projectName!]!;
 
-  const packageRoot = joinPathFragments(
```

Comment:
The comment on the `joinPathFragments` function explicitly states not to use it for building paths to read or write files on disk due to it removing windows letter drive prefixes. I've updated this to just use plain `join(...)` instead.